templates/prioritization_agenda.tt

```diff
@@ -28,9 +29,9 @@ note_id: xxx
 - Old MCPs (not seconded, take a look)
 {{-issues::render(issues=mcp_old_not_seconded, indent="  ", with_age=true, empty="No old proposals this time.")}}
 - Pending FCP requests (check your boxes!)
-{{-issues::render(issues=in_pre_fcp, indent="  ", empty="No pending FCP requests this time.")}}
+{{-issues_rfcbot::render(issues=in_pre_fcp, indent="  ", empty="No pending FCP requests this time.")}}
 - Things in FCP (make sure you're good with it)
```

Comment:
@Urgau The items in this section lost the links to the links to the pull requests

probably worth checking if your new macro is not eating other links around

Reply:
Sorry for that, based on the extra information you gave in DM, I was able to fix it.

To make in short, when we don't have any details for an FCP we use a fallback and that fallback was manual instead of using the `_issue.tt` template 🤦

I pushed a commit using the proper fallback *as well as fixing a weird newline issue*.